meta-mel-support/recipes-bsp/96boards-tools/96boards-tools_0.7.bb

```diff
@@ -26,7 +27,13 @@ do_install () {
 
     install -d ${D}${sbindir}
     install -m 0755 ${S}/resize-helper ${D}${sbindir}
+
+	install -d ${D}${sysconfdir}/init.d
+	install -m 0755 ${WORKDIR}/resize-helper.sh ${D}${sysconfdir}/init.d/resize-helper.sh
```

Comment:
This script hardcodes paths, both the binary paths used in PATH and the hardcoded /etc. You should add placeholders and replace them with sed, and avoid relying on PATH to run resize-helper, run it explicitly in the path where it was installed.